discovery-provider/src/queries/get_tracks.py

```diff
@@ -117,6 +143,24 @@ def get_tracks_and_ids():
                 )
                 args["user_id"] = user_id
 
+            if "routes" in args:
+                # Convert the handles to user_ids
+                routes = args.get("routes")
+                handles = [route["handle"].lower() for route in routes]
+                user_id_tuples = (
+                    session.query(User.user_id, User.handle_lc)
+                    .filter(User.handle_lc.in_(handles), User.is_current == True)
+                    .all()
+                )
+                user_id_map = {handle: user_id for (user_id, handle) in user_id_tuples}
+                args["routes"] = [
+                    {
+                        "slug": route["slug"],
+                        "owner_id": user_id_map[route["handle"].lower()],
```

Comment:
I think it's possible here that `route["handle"].lower()` doesn't exist in `user_id_map` (e.g. i pass in a non-existent handle as api request arg) in which case this line will error